src/openalea/phenomenal/segmentation/plane_interception.py

```diff
@@ -11,7 +11,11 @@
 
 import math
 import numpy
-import networkx
+
+try:
```

Comment:
nx_cugraph is not in the dependencies. It would be nice to add it at least with an optional [gpu] in project.toml

Reply:
Agree that it should be mentionned somewhere ! Not sure about the inclusion in pyprojet, due to its install specificity/nvidia requirement. 
But may be a paragrah in in readme can do the job:

'The package supports gpu acceleration of networkx by nx_cugraph. If you have nvidia gpu, just add it in your environment: 

conda install -c rapidsai -c conda-forge -c nvidia nx-cugraph